wally-registry-backend/Cargo.toml

```diff
@@ -23,6 +23,8 @@ git2 = "0.13.17"
 reqwest = { version = "0.11.0", features = ["blocking", "json"] }
 rocket = { git = "https://github.com/SergioBenitez/Rocket", rev = "2893ce754d6535e0a752586e60d7e292343016c0" }
 rocket_contrib = { git = "https://github.com/SergioBenitez/Rocket", rev = "2893ce754d6535e0a752586e60d7e292343016c0" }
+rusoto_core = "0.48.0"
+rusoto_s3 = "0.48.0"
```

Comment:
Could you split these deps into feature flags? one for S3, one for GCP (which is on by default)?

Reply:
I can definitely do that - the backend doesn't have any existing flags though so I would like to hear what the maintainers think about adding feature flags as well and if it is in line with what they want

Reply:
I think adding feature flags for this is a good idea!